testing/sha_comp_support_GenerateTables_test.go

```diff
@@ -0,0 +1,25 @@
+package testing_test
+
+import (
+	"fmt"
+	. "github.com/pegnet/LXRHash"
+	"testing"
+)
+
+// Create all the tables for a range of parameters for the LXRHash
+// These tables are used to test and characterize the LXRHash
+
+// TestGenerateTables()
+// Generate all the map files for a particular seed, and number of passes
+func TestGenerateTables(t *testing.T) {
+
+	// Create all size tables with a given seed, up to 10 passes and for sizes upto 4 GB
```

Comment:
If the test goes up to 4gb, this might blow any automated building in the repi (travis/circle)